Website/src/rendering/src/components/Sponsors/SponsorInformationPageHero.tsx

```diff
@@ -0,0 +1,39 @@
+import { Field, ImageField } from '@sitecore-jss/sitecore-jss-nextjs';
+import { ComponentProps } from 'lib/component-props';
+import InformationPageHero from '../NonSitecore/InformationPageHero';
+
+export type SponsorInformationPageHeroProps = ComponentProps & {
+  fields: {
+    Name: Field<string>;
+    Logo: ImageField;
+    Level: Field<string>;
+    FacebookProfileLink?: Field<string>;
+    TwitterProfileLink?: Field<string>;
+    InstagramProfileLink?: Field<string>;
+    LinkedinProfileLink?: Field<string>;
+  };
+};
+
+const SponsorInformationPageHero = (props: SponsorInformationPageHeroProps): JSX.Element => {
+  const { fields, ...propsRest } = props;
+  const newFields = {
+    Name: fields.Name,
+    Image: fields.Logo,
+    FacebookProfileLink: fields.FacebookProfileLink,
+    TwitterProfileLink: fields.TwitterProfileLink,
+    InstagramProfileLink: fields.InstagramProfileLink,
+    LinkedinProfileLink: fields.LinkedinProfileLink,
+  };
+  const qualificative = props.fields.Level?.value ? props.fields.Level.value : '';
+
+  return (
+    <InformationPageHero
+      {...propsRest}
+      fields={newFields}
+      type="sponsor"
+      qualificative={qualificative}
```

Comment:
The qualificative/level for sponsors seems to be "platinum", "gold" or "silver" by looking at the various sponsor pages.

Some of them should be displayed in orange instead of blue. This is handled in `InformationPageHero.css` on line 112-113. We must add `.platinum` to that list of classes which will display in orange. Please validate with Scott/Todor whether we also want to add ".gold" or ".silver" to the list as well.